packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -585,9 +585,7 @@ - (FLTTextureMessage*)create:(FLTCreateMessage*)input error:(FlutterError**)erro
 //                                                                 .alphanumericCharacterSet];
 
       player = [[FLTVideoPlayer alloc] initWithURL:[NSURL URLWithString:input.uri]
-                                      frameUpdater:frameUpdater
-                                        enableCache:enableCache
```

Comment:
Do we need to hardcode enable the cache? If there's a parameter, I think we should use it. Or is there a problem with that?